--- a/tests/runtime_shell/CMakeLists.txt	2023-12-15 13:55:20.527844590 +0000
+++ b/tests/runtime_shell/CMakeLists.txt	2023-12-15 13:57:28.525757818 +0000
@@ -7,11 +7,6 @@
   in_dummy_expect.sh
   in_tail_expect.sh
   in_http_tls_expect.sh
-  in_syslog_tcp_tls_expect.sh
-  in_syslog_tcp_plaintext_expect.sh
-  in_syslog_udp_plaintext_expect.sh
-  in_syslog_uds_dgram_plaintext_expect.sh
-  in_syslog_uds_stream_plaintext_expect.sh
   )
 
 # Prepare list of unit tests
